content/chainguard/libraries/java/build-configuration.md

```diff
@@ -63,6 +63,13 @@ Follow the steps from the [global
 configuration](/chainguard/libraries/java/global-configuration/#sonatype-nexus-repository)
 to determine URL and authentication details.
 
+## Direct access
```

Comment:
Just flagging that I'm a little iffy on this whole doc. Admittedly I'm not very familiar with Libraries, but I'm a little unclear on what each section is telling me.

As an example, each section starts with `Build configuration to retireve artifacts ....`. These all feel like incomplete sentences and I'm not sure what they're supposed to mean.